src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -261,4 +282,23 @@ public ValidationResult validateMsg(String msgType, JsonObject jsonvalidateMessa
         }
         return validationResult;
     }
+    
+    /**
+	 * This method is used to override given input semantics gav or to set if not exist 
```

Comment:
little modification on 
This method is used to override given input meta.source.serializer GAV with semantics GAV, If meta.source.serializer GAV not existed it creates the semantics GAV as meta.source.serializer GAV